lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp

```diff
@@ -357,6 +357,20 @@ ResolveTypeAlias(SwiftASTContext *module_holder,
                  swift::Demangle::NodePointer node,
                  bool prefer_clang_types = false) {
   LLDB_SCOPED_TIMER();
+  auto resolve_clang_type = [&]() {
+    if (node->getNumChildren() == 2 && node->getChild(0)->hasText() &&
+        node->getChild(0)->getText() == swift::MANGLING_MODULE_OBJC &&
+        node->getChild(1)->hasText()) {
```

Comment:
should we check node kinds here? Specifically, check that:

1. first child has kind `Module`
2. second child has kind `Identifier`

Reply:
There's a helper
```
/// Return a pair of modulename, type name for the outermost nominal type.
llvm::Optional<std::pair<StringRef, StringRef>>
GetNominal(swift::Demangle::Demangler &dem, swift::Demangle::NodePointer node) {
```
on line 1212.